SwrveSDK/src/amazon/java/com/swrve/sdk/ISwrvePushNotificationListener.java

```diff
@@ -0,0 +1,18 @@
+package com.swrve.sdk;
+
+import android.os.Bundle;
+
+/**
+ * Implement this interface to be notified of any Swrve push notification
+ * to your app.
+ */
+public interface ISwrvePushNotificationListener {
```

Comment:
Yeah, not a bad idea but didn't do because the google one resides in package com.swrve.sdk.gcm.